.github/workflows/release-base.yml

```diff
@@ -21,15 +21,17 @@ jobs:
       run: pip install --disable-pip-version-check --upgrade pip setuptools wheel
 
     - name: Install ddev
-      run: pip install -e ./datadog_checks_dev[cli]
+      run: |
+        pip install -e ./datadog_checks_dev[cli]
+        pip install -e ./ddev
 
     - name: Configure ddev
       run: |
         ddev config set repos.core .
         ddev config set repo core
 
-    - name: Release ddev
-      run: ddev release upload datadog_checks_base
+    - name: Release base package
+      run: ddev release upload -s datadog_checks_base
```

Comment:
Just wondering what does `-s` mean? I saw it's a flag for `upload` but we don't have a description for it: https://datadoghq.dev/integrations-core/ddev/cli/#ddev-release-upload

Reply:
Build sdist too